controllers/binding/snapshotenvironmentbinding_adapter_test.go

```diff
@@ -501,17 +503,33 @@ var _ = Describe("Binding Adapter", Ordered, func() {
 				ContextKey: loader.ApplicationContextKey,
 				Resource:   hasApp,
 			},
+			{
+				ContextKey: loader.ComponentContextKey,
+				Resource:   hasComp,
+			},
 			{
 				ContextKey: loader.EnvironmentContextKey,
 				Resource:   hasEnv,
 			},
+			{
+				ContextKey: loader.SnapshotContextKey,
+				Resource:   hasSnapshot,
+			},
 			{
 				ContextKey: loader.DeploymentTargetContextKey,
 				Resource:   deploymentTarget,
 			},
+			{
+				ContextKey: loader.DeploymentTargetClaimContextKey,
+				Resource:   deploymentTargetClaim,
+			},
+			{
+				ContextKey: loader.SnapshotEnvironmentBindingContextKey,
+				Resource:   hasBinding,
+			},
 		})
 
-		dtc, _ := adapter.loader.GetDeploymentTargetClaimForEnvironment(k8sClient, adapter.context, hasEnv)
+		dtc, _ = adapter.loader.GetDeploymentTargetClaimForEnvironment(k8sClient, adapter.context, hasEnv)
```

Comment:
You don't need to declare `dtc` above, can just keep `:=`, but it doesn't matter. LGTM